microprofile/WFLY-19954_Preview_Support_vertx_feature_pack.adoc

```diff
@@ -0,0 +1,232 @@
+| warningExceptionTimeUnit | TimeUnit | {vertx_options_with_link} | warning-exception-time-unit | the time unit for warningExceptionTime
+| classPathResolvingEnabled | boolean | {filesystem_options_with_link} | classpath-resolving-enabled | whether classpath resolving is enabled
+| fileCachingEnabled | boolean | {filesystem_options_with_link} | file-cache-enabled | whether file caching is enabled for class path resolving
+| fileCacheDir | String | {filesystem_options_with_link} | file-cache-dir | file cache directory, defaults to `${jboss.server.temp.dir}/vertx-cache`
+| addressResolverOptions | AddressResolverOptions | {vertx_options_with_link} | address-resolver-option | reference to the AddressResolverOptions name
+
+|===
+
+Options in {address_resolver_options_with_link} are grouped to the address `/subsystem=vertx/address-resolver-option=xxx`, this can also be merged into `/subsystem=vertx/vertx-option=xxx` too if that is more user-friendly.
+
+.Attributes for AddressResolverOptions in address: `/subsystem=vertx/address-resolver-option=xxx`
+[frame=all, grid=all]
+|===
+^| Field Name ^| Data Type ^| Source Class ^| Resource Attribute Name ^| Note
+
+| hostsValue | Buffer | {address_resolver_options_with_link} | hosts-value | the hosts configuration file value
+| hostsRefreshPeriod | int | {address_resolver_options_with_link} | hosts-refresh-period | the hosts configuration refresh period in millis
+| servers | List<String> | {address_resolver_options_with_link} | servers | list of DNS servers
+| optResourceEnabled | boolean | {address_resolver_options_with_link} | opt-resource-enabled | if an optional record is automatically included in DNS queries
+| cacheMinTimeToLive | int | {address_resolver_options_with_link} | cache-min-time-to-live | the cache min TTL in seconds
+| cacheMaxTimeToLive | int | {address_resolver_options_with_link} | cache-max-time-to-live | the cache max TTL in seconds
+| cacheNegativeTimeToLive | int | {address_resolver_options_with_link} | cache-negative-time-to-live | the cache negative TTL in seconds
+| queryTimeout | long | {address_resolver_options_with_link} | query-time-out | the query timeout in milliseconds
+| maxQueries | int | {address_resolver_options_with_link} | max-queries | the maximum number of queries to be sent during a resolution
+| rdFlag | boolean | {address_resolver_options_with_link} | rd-flag | the DNS queries Recursion Desired flag value
+| searchDomains | List<String> | {address_resolver_options_with_link} | search-domains | the list of search domains
+| ndots | int | {address_resolver_options_with_link} | n-dots | the ndots value
+| rotateServers | boolean | {address_resolver_options_with_link} | rotate-servers | if the dns server selection uses round-robin
+| roundRobinInetAddress | boolean | {address_resolver_options_with_link} | round-robin-inet-address | if the inet address selection uses round-robin
```

Comment:
Same comment as above re the attribute names in a 'community' stability variant:

s/hostsValue/hosts-value